File: src/main/java/plus/maa/backend/controller/response/copilot/CopilotPageInfo.java
package plus.maa.backend.controller.response.copilot;

import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Data;
import lombok.experimental.Accessors;

import java.io.Serializable;
import java.util.List;

/**
 * @author LoMu
 * Date  2022-12-27 12:39
 */
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
@Accessors(chain = true)
@Data
public class CopilotPageInfo implements Serializable {
    private Boolean hasNext;
    private Integer page;
    private Long total;
    private List<CopilotInfo> data;
}

File: src/main/java/plus/maa/backend/controller/response/copilot/CopilotPageInfo.kt
package plus.maa.backend.controller.response.copilot

import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming
import java.io.Serializable

/**
 * @author LoMu
 * Date  2022-12-27 12:39
 */
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
data class CopilotPageInfo(
    private val hasNext: Boolean,
    private val page: Int,
    private val total: Long,
    private val data: List<CopilotInfo>
) : Serializable

